package.json

```diff
@@ -120,6 +120,23 @@
             "default": true,
             "description": "%configuration.SAS.session.outputHtml%"
           },
+          "SAS.results.html.style": {
```

Comment:
I'm going to submit another PR after this one to change "SAS.session.outputHtml" to "SAS.results.html.enabled" to follow the pattern introducing in this pattern. "results" makes a lot more sense than "session" here, and this new pattern makes potentially adding additional ODS results types in the future easier. Also re-ordering the options in that follow-on PR, so that the connection profiles and certs come first, then the results related options.